pkg/slo-controller/noderesource/framework/extender_plugin.go

```diff
@@ -24,8 +24,17 @@ import (
 	"github.com/koordinator-sh/koordinator/pkg/slo-controller/metrics"
 )
 
+// node resource initialization:
```

Comment:
need more explanation about the running order of these plugins

e.g. there are several stages:
Setup, Reset if node not enabled

Setup
Calculate(run plugins for generate internal nr)
Update(run PreUpdatePlugin for nr injection or other crd update; run checkNeedSync plugin, run Prepare plugins, do update node obj)